cli/command/volume/create.go

```diff
@@ -29,16 +30,33 @@ func newCreateCommand(storageosCli *command.StorageOSCli) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:   "create [OPTIONS] [VOLUME]",
+		Use:   "create [OPTIONS] [NAMESPACE]/[VOLUME]",
 		Short: "Create a volume",
 		Args:  cli.RequiresMaxArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if len(args) == 1 {
-				if opt.name != "" {
-					fmt.Fprint(storageosCli.Err(), "Conflicting options: either specify --name or provide positional arg, not both\n")
+				switch {
+				case opt.name != "" && opt.namespace != "":
+					fmt.Fprint(storageosCli.Err(), "Conflicting options: (--namespace, --name and positional NAMESPACE/VOLUME formats used) These are mutualy exclusive\n")
 					return cli.StatusError{StatusCode: 1}
+
+				case opt.name != "":
+					fmt.Fprint(storageosCli.Err(), "Conflicting options: (--name and positional NAMESPACE/VOLUME formats used) These are mutualy exclusive\n")
+					return cli.StatusError{StatusCode: 1}
+
+				case opt.namespace != "":
+					fmt.Fprint(storageosCli.Err(), "Conflicting options: (--namespace and positional NAMESPACE/VOLUME formats used) These are mutualy exclusive\n")
+					return cli.StatusError{StatusCode: 1}
+
+				default:
+					// Parse the user input to get namespace and volume name
+					namespace, name, err := storageos.ParseRef(args[0])
+					if err != nil {
+						return err
+					}
+					opt.name = name
+					opt.namespace = namespace
```

Comment:
Not sure I understand the logic in this block.  The following args should work:
-n ns1 vol1
ns1/vol1
vol1  (result: default/vol1)
-n ns1 --name vol1
--namespace ns1 --name vol1

I'm pretty sure the default namespace won't work?

Would suggest moving this validation/parsing into a separate function and adding tests.